Add currency icons

johnletey opened the pull request:
Closes #182

locness3 commented:
You shoudl rename your PR to "feat: Add currency and banknote icons"

mittalyashu commented:
## Banknote

To me, it doesn't really look like a banknote _(sorry)_

| Icon |
| --- |
| ![image](https://user-images.githubusercontent.com/29014463/61594492-33c96d80-ac0a-11e9-837f-6dbed69a625c.png) |

## Bitcoin

In bitcoin logo the second curve should not be far, instead, it should be of the same size and tilted to 10 deg of the angle.

| Icon | Original |
| --- | --- |
| ![Figma_1MStwoFT6u](https://user-images.githubusercontent.com/29014463/61594474-f49b1c80-ac09-11e9-94be-d30ce4e015e5.png) | ![image](https://user-images.githubusercontent.com/29014463/61594403-092ae500-ac09-11e9-8ed1-9aa080494df9.png) |

## Swiss Franc

In the first of the F, there is a lot of empty space in between

| Icon | Example |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/29014463/61594420-3e373780-ac09-11e9-9d90-386fced7063c.png) | ![image](https://user-images.githubusercontent.com/29014463/61594527-9ae72200-ac0a-11e9-9eb8-a4f9cd4f72ef.png) |

## Euros

Add the ends of euro looks too close

| Icon | Example |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/29014463/61594367-d08b0b80-ac08-11e9-99c0-0d594f2373dc.png) | ![image](https://user-images.githubusercontent.com/29014463/61594553-d08c0b00-ac0a-11e9-8f4a-5e5da3b78ee6.png) |

## GBP

In GBP icon, instead of keeping it flat straight, I think there should be a curve

| Icon | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/29014463/61594446-7b032e80-ac09-11e9-903f-157f91abb866.png) | ![image](https://user-images.githubusercontent.com/29014463/61594576-fe714f80-ac0a-11e9-8d6d-3421558a2cd9.png) |

## Rub

Maybe try decreasing the height of this area

| Icon | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/29014463/61594462-d3d2c700-ac09-11e9-9810-69edb50d3b39.png) | ![image](https://user-images.githubusercontent.com/29014463/61594594-2f518480-ac0b-11e9-80a9-db6628d02105.png) |

johnletey commented:
@locness3 @mittalyashu

> ## Banknote
> To me, it doesn't really look like a banknote _(sorry)_
> 
> Icon
> ![image](https://user-images.githubusercontent.com/29014463/61594492-33c96d80-ac0a-11e9-837f-6dbed69a625c.png)

I'm open for suggestions!

> ## Bitcoin
> In bitcoin logo the second curve should not be far, instead, it should be of the same size and tilted to 10 deg of the angle.
> 
> Icon	Original
> ![Figma_1MStwoFT6u](https://user-images.githubusercontent.com/29014463/61594474-f49b1c80-ac09-11e9-94be-d30ce4e015e5.png)	![image](https://user-images.githubusercontent.com/29014463/61594403-092ae500-ac09-11e9-8ed1-9aa080494df9.png)

Like this?

![Shot 2019-07-22 at 18 53 35](https://user-images.githubusercontent.com/30328854/61653383-1223c100-acb2-11e9-9101-0ee4952be8b7.jpg)

> ## Swiss Franc
> In the first of the F, there is a lot of empty space in between
> 
> Icon	Example
> ![image](https://user-images.githubusercontent.com/29014463/61594420-3e373780-ac09-11e9-9d90-386fced7063c.png)	![image](https://user-images.githubusercontent.com/29014463/61594527-9ae72200-ac0a-11e9-9eb8-a4f9cd4f72ef.png)

Have updated so that the second line of the F is exactly in the middle of the other two lines:

![Shot 2019-07-22 at 18 44 47](https://user-images.githubusercontent.com/30328854/61652880-e48a4800-acb0-11e9-9f55-2a7cac30a415.jpg)

> ## Euros
> Add the ends of euro looks too close
> 
> Icon	Example
> ![image](https://user-images.githubusercontent.com/29014463/61594367-d08b0b80-ac08-11e9-99c0-0d594f2373dc.png)	![image](https://user-images.githubusercontent.com/29014463/61594553-d08c0b00-ac0a-11e9-8f4a-5e5da3b78ee6.png)

Updated:

![Shot 2019-07-22 at 18 47 56](https://user-images.githubusercontent.com/30328854/61653024-434fc180-acb1-11e9-9b96-a44e872065c1.jpg)

> ## GBP
> In GBP icon, instead of keeping it flat straight, I think there should be a curve
> 
> Icon	Original
> ![image](https://user-images.githubusercontent.com/29014463/61594446-7b032e80-ac09-11e9-903f-157f91abb866.png)	![image](https://user-images.githubusercontent.com/29014463/61594576-fe714f80-ac0a-11e9-8d6d-3421558a2cd9.png)

Updated:

![Shot 2019-07-22 at 18 42 59](https://user-images.githubusercontent.com/30328854/61652745-937a5400-acb0-11e9-8f77-537d7761fbfa.jpg)

> ## Rub
> Maybe try decreasing the height of this area
> 
> Icon	Original
> ![image](https://user-images.githubusercontent.com/29014463/61594462-d3d2c700-ac09-11e9-9810-69edb50d3b39.png)	![image](https://user-images.githubusercontent.com/29014463/61594594-2f518480-ac0b-11e9-80a9-db6628d02105.png)

Updated:

![Shot 2019-07-22 at 18 55 08](https://user-images.githubusercontent.com/30328854/61653466-4303f600-acb2-11e9-96fc-9bd8d04159f5.jpg)

mittalyashu commented:
## Bitcoin

The tile of the bitcoin looks good, can you also try a newer version of bitcoin in which size of both the bumps is same.

![image](https://user-images.githubusercontent.com/29014463/61706507-e235f580-ad65-11e9-8472-5dd3178ca6b2.png)

## GBP

We need to work on that bumpy curve 😅

![image](https://user-images.githubusercontent.com/29014463/61706490-d77b6080-ad65-11e9-8de0-94f7ae54e3c6.png)

## Swiss Franc, Euros, Rub

Now, it looks good.

johnletey commented:
> ## Bitcoin
> The tile of the bitcoin looks good, can you also try a newer version of bitcoin in which size of both the bumps is same.
> 
> ![image](https://user-images.githubusercontent.com/29014463/61706507-e235f580-ad65-11e9-8472-5dd3178ca6b2.png)

@mittalyashu How is this?

![Shot 2019-07-23 at 14 53 03](https://user-images.githubusercontent.com/30328854/61717819-aeed6980-ad59-11e9-969e-4781e612db10.jpg)

> ## GBP
> We need to work on that bumpy curve 😅
> 
> ![image](https://user-images.githubusercontent.com/29014463/61706490-d77b6080-ad65-11e9-8de0-94f7ae54e3c6.png)

Maybe give me an example?

> ## Swiss Franc, Euros, Rub
> Now, it looks good.

Updated the PR

mittalyashu commented:
Bitcoin icon looks good now.

Let me find an example of the GBP icon.

locness3 commented:
Maybe adding a bitcoin tilted icon too ?

MarcelloTheArcane commented:
I think the BTC icon should be straight by default - is that possible?

mittalyashu commented:
No @MarcelloTheArcane it's not straight, you can check the official bitcoin website.

johnletey commented:
@locness3 @mittalyashu @MarcelloTheArcane Sorry ... didn't ever update the `btc` icon

It's now updated ... just need to work on the GBP

antony commented:
Excited about the GBP icon!

locness3 commented:
@colebemis We're still waiting for you...

J0XX0 commented:
This repository badly needs some extra moderators / pr-reviewers.

locness3 commented:
Could someone reach out to @colebemis on Twitter?
